### PR TITLE
chore!: upgrade to lodash v4

### PR DESCRIPTION
**Pull request by taopkorczak**
Related: https://oat-sa.atlassian.net/browse/ADF-1579

Update lodash methods to version 4.
Documentation: https://github.com/lodash/lodash/wiki/Migrating

### CODE REVIEW

**Comment by oatymart** [comment]
Should something be done in the following places?
- https://github.com/oat-sa/tao-core/blob/develop/views/package.json#L13  tao-core-libs dependency to [1.0.0](https://github.com/oat-sa/tao-core-libs-fe/releases/tag/v1.0.0)
- https://github.com/oat-sa/tao-core/blob/develop/views/package.json#L14 tao-core-sdk dependency to [3.0.0](https://github.com/oat-sa/tao-core-sdk-fe/releases/tag/v3.0.0)
- https://github.com/oat-sa/tao-core/blob/develop/views/package.json#L16 tao-core-ui dependency to [3.0.0](https://github.com/oat-sa/tao-core-ui-fe/releases/tag/v3.0.0)
- https://github.com/oat-sa/tao-core/blob/develop/views/package.json#L23 lodash dependency to 4.x

Reason: I want to test the extension branches, knowing that lodash 4 is the only one installed and is being accessed from now on.

**Comment by poyuki** [comment]
one silly question from my side didn't required update of lodash version there https://github.com/oat-sa/tao-core/blob/feature/ADF-1579/update-lodash-to-4/views/package.json#L23?

**Review by poyuki — COMMENTED** [review]
https://github.com/oat-sa/tao-core/pull/3933#issuecomment-1860093172

File: views/package.json [hunk 1]
```diff
@@ -10,17 +10,17 @@
...
-    "@oat-sa/tao-core-ui": "^2.2.0",
+    "@oat-sa/tao-core-ui": "^3.0.0",
     "codemirror": "^5.54.0",
     "dompurify": "^2.4.0",
     "gamp": "0.2.1",
     "handlebars": "1.3.0",
     "interactjs": "1.3.4",
     "jquery": "1.9.1",
-    "lodash": "2.4.1",
+    "lodash": "4.17.21",
```
Comment (poyuki) [inline views/package.json:23]: I suppose the same lodash should be applied to other extensions.
Also I suppose package-lock.json also need to be update?

**Pull request merged**